src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -83,24 +83,14 @@ AudioStreamAAudio::AudioStreamAAudio(const AudioStreamBuilder &builder)
     isSupported();
 }
 
-AudioStreamAAudio::~AudioStreamAAudio()
-{
+AudioStreamAAudio::~AudioStreamAAudio() {
     delete[] mFloatCallbackBuffer;
     delete[] mShortCallbackBuffer;
 }
 
-static bool isRunningOnAndroid8_1OrHigher() {
-#ifdef __ANDROID__
-  char sdk[PROP_VALUE_MAX] = {0};
-  if (__system_property_get("ro.build.version.sdk", sdk) != 0) {
-      return atoi(sdk) >= 27;  // SDK Level 27 is Android Oreo 8.1
-  }
-#endif
-  return false;
-}
-
 bool AudioStreamAAudio::isSupported() {
-    if (!isRunningOnAndroid8_1OrHigher()) {
+    const int SDK_8_1 = 27; // OC-MR1
```

Comment:
Please rename to "ANDROID_8_1". People usually associate "SDK" with API versions, so "SDK 8.1" looks strange.

Reply:
Done